src/vs/workbench/api/common/extHostTreeViews.ts

```diff
@@ -497,12 +497,12 @@ class ExtHostTreeView<T> extends Disposable {
 
 	private getHandlesToRefresh(elements: T[]): TreeItemHandle[] {
 		const elementsToUpdate = new Set<TreeItemHandle>();
```

Comment:
With your change, I'm not sure we need this part anymore:

https://github.com/alanrenmsft/vscode/blob/4499290f09b1e879958dab21bd1f4bc9521655b9/src/vs/workbench/api/common/extHostTreeViews.ts#L517-L518